src/tests.rs

```diff
@@ -66,10 +66,12 @@ author : yaml_supporter
 args:
     - apple : 
         short: a
-        value_parser: non_empty_string
+        value_parser: 
+            type: non_empty_string
     - banana:
         short: b
         long: banana
+        value_parser: non_empty_string
```

Comment:
could you add tests for numeric value parsers? the schema isn't too clear to me

Reply:
I've added a numeric value parser test.
https://github.com/aobatact/clap-serde/blob/003b92eb3016892772fc72c6a63e912f1e0ad5db/src/tests.rs#L58-L92

Reply:
Thanks

I would lean towards just forcing it to be inclusive  This will line up with what I'd recommend for clap v4's `num_args`.  Up to you.

Reply:
I changed the range of values to be inclusive by default.